extensions/ql-vscode/src/model-editor/methods-usage/methods-usage-data-provider.ts

```diff
@@ -253,27 +246,9 @@ function urlValueResolvablesAreEqual(
   return false;
 }
 
-function sortMethodsInGroups(
-  methods: readonly Method[],
-  modeledMethods: Readonly<Record<string, readonly ModeledMethod[]>>,
-  mode: Mode,
-  modifiedMethodSignatures: ReadonlySet<string>,
-  processedByAutoModelMethods: ReadonlySet<string>,
-): Method[] {
+function sortMethodsInGroups(methods: readonly Method[], mode: Mode): Method[] {
```

Comment:
Perhaps name this something like `createGroups` or `groupMethods` since we're not actually sorting the methods?

Reply:
We can't really call it `groupMethods` because that's also the name of the method in `sorting.ts`. I've renamed it to `createGroups` and we can always tweak again in the future if we want to.